### ISSUE TITLE
Configure Javadoc for Extensions without sources

### ISSUE THREAD

**Issue by fibsifan** [epam/sap-commerce-intellij-idea-plugin#1026]
Hello,

with my newly gained understanding of module configuration for SAP CC Modules that include sourcecode (#1025) I was wondering if the Javadoc-Feature is working correctly?

I'm on Version 2023.3.5 and imported a project without preexisting .idea configuration (I deleted the .idea and idea-module-files folder of my project and did a fresh import).

I tried this twice. The first time, I left the Javadoc-URL as is, the second time I took the URL for 2211.12 from [the overview page on help.sap.com](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/12be4ac419604b01aabb1adeb2c4c8a2/179bbc9b35274d7ca784e46b3beb40b2.html?locale=en-US).

In both cases, the quick-help for classes from modules without sources didn't include the javadocs from the internet (I tried de.hybris.platform.couponservices.services.CouponService which [definately has javadoc in the hybris javadoc](https://help.sap.com/doc/15fe6db3c7dc4f30a5c378eb137f49ac/2211.12/en-US/de/hybris/platform/couponservices/services/CouponService.html).

![grafik](https://github.com/epam/sap-commerce-intellij-idea-plugin/assets/574988/5e1a2942-8fd7-41f1-9f4d-327b9877c31b)

After manually adding the javadoc URL to the `couponservices - server` library, the quick-help (shortcut is Ctrl+q with default IntelliJ keyboard layout) did show up.

![grafik](https://github.com/epam/sap-commerce-intellij-idea-plugin/assets/574988/a349f214-1e5a-4222-9db1-0ab652557afd)

Couldn't the plugin automatically configre the hybris Javadoc URL for all modules, that have a ` - server` library defined? It seems to me that the modules without a `- server` library seem to be the ones that have sources.

**Comment by fibsifan** [comment]
I also also noted that the integration extension pack has a separate javadoc URLs. But I didn't find any extension in the integration extension package that doesn't have sources included, so this might not be relevant.

**Comment by mlytvyn** [comment]
@fibsifan , thank you for sharing this, looks like regression :)

**Comment by StefanKruk** [comment]
@mlytvyn here is the fix for this issue: https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/1027

**Comment by fibsifan** [comment]
Hi @mlytvyn , @StefanKruk 

I'm not sure if I'm doing anything wrong. I did a Project refesh as well as a fresh project import and still no javadoc. When I add it manually - for example to "adaptivesearch - server" then javadocs get fetched. 

I noticed, that there are sources configured for adaptivesearch - server but that are only the testsrc: 
![grafik](https://github.com/epam/sap-commerce-intellij-idea-plugin/assets/574988/404fa3d2-e040-474b-b0e3-98c9bcafad4c)

Edit: I'm now at IntelliJ version 2024.1 and the Plugin has version 2024.1.0
